doc/write_your_own/components/CoordSlider.ipynb

```diff
@@ -1,95 +0,0 @@
-{
```

Comment:
Because it was irrelevant.  It was an example of how to make a slider, which will be constructed in Cube Browser anyway.